Source/OpenTK/Platform/Egl/EglContext.cs

```diff
@@ -97,9 +95,7 @@ abstract class EglContext : EmbeddedGraphicsContext
                 window.CreateWindowSurface(config);
 
             int[] attrib_list = new int[] { Egl.CONTEXT_CLIENT_VERSION, major, Egl.NONE };
-            HandleAsEGLContext = Egl.CreateContext(window.Display, config, shared != null ? shared.HandleAsEGLContext : IntPtr.Zero, attrib_list);
-
-            MakeCurrent(window);
+            HandleAsEGLContext = Egl.CreateContext(window.Display, config, sharedContext != null ? (sharedContext as IGraphicsContextInternal).Context.Handle : IntPtr.Zero, attrib_list);
```

Comment:
Ah, good point. As long as it's no-worse than our current ones, It's okay as-is.

If you want to, you can fix them, but it shouldn't be necessary for us to merge this PR.